### ISSUE TITLE
When I try training on my laptop ,some questions raised

### ISSUE THREAD

**Issue by ming71** [open-mmlab/mmdetection#502]
Firstly, the picture blow shows that network structure misfits the pre-trained model parameter . I didn't change any module in mask rcnn network , are there some dismatches between pre-trained model and mask rcnn network?
![image](https://user-images.githubusercontent.com/43720092/56208948-fabab380-6084-11e9-81a5-4ae761c64112.png)
Secondly , code here seem to  meet some problem : there no key named `text` in cfg as well as cfg.checkpoint_config.meta , anything wrong to my operations?
```
        cfg.checkpoint_config.meta = dict(
            mmdet_version=__version__, config=cfg.text)
``` 
https://github.com/open-mmlab/mmdetection/blob/53c647eac839aac788d719ba2c5f8215214cbdd8/tools/train.py#L57-L58

**Comment by hellock** [comment]
1. `num_batches_tracked` is added in new versions pytorch but they did not update the pretrained models, so this is normal.
2. `cfg` is not a dict but ConfigDict object.
